src/misc/cryptocurrencies.ts

```diff
@@ -0,0 +1,50 @@
+const cryptocurrencies: { [key: string]: Cryptocurrency } = {
```

Comment:
this file has been renamed to `chains.ts`, same reason as above

Reply:
removed `cryptocurrencies.ts` in favor of `chains.ts`